attbillsplitter/main.py

```diff
@@ -141,16 +141,35 @@ def login(self):
             return True
         else:
             if 'promo' in login_submit.url.lower():
-                print ('\U0001F534  Popup window detected during login. '
+                reject_status = self.clickSkipPromo()
+
+                if not reject_status:
```

Comment:
no need to define `reject_status` above, just do `if not self.click_skip_promo():